src/pages/Jobs.js

```diff
@@ -1,6 +1,6 @@
 import React ,{useState,useEffect}from 'react'
 import '../App.css';
-import JobList from '../components/JobList'
+import JobCard from '../components/JobCard'
 import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
not necessary

Reply:
i removed taht ../App.css and bootstrap part as well